specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -4851,6 +4856,24 @@
       ],
       "type": "object"
     },
+    "ImageMetadata": {
+      "type": "object",
+      "description": "Returns metadata about the operating system image for this compute instance.",
+      "properties": {
+        "currentImageVersion": {
+          "type": "string",
+          "description": "Specifies the current operating system image version this compute instance is running on."
+        },
+        "latestImageVersion": {
```

Comment:
should this and currentImageVersion be readOnly?

Reply:
We've got a partner request from Defender for Cloud to make this one patchable. ReadOnly should be good for now, but in case we want to make it patchable in the future, would that be possible later?

Reply:
top level property "osImageMetadata" referring to ImageMetadata is itself a read only property.. Shouldn't that suffice